src/CMakeLists.txt

```diff
@@ -433,7 +433,7 @@ install(
 
 if(${WITH_LTTNG})
   add_subdirectory(tracing)
-  add_dependencies(common-objs oprequest-tp)
+  add_dependencies(common-objs tracepoint_libraries)
```

Comment:
I think the problem is that the only target which builds them otherwise is vstart.  It seemed simpler to do it here rather than to add the specific -tp targets to each place.

Reply:
> only target which builds them otherwise is vstart

yes. but what the problem/pain point this change is trying to address?

Reply:
Ok, I think it was left over from before I went through and added in the EVENTTRACE dependencies everywhere else.  I've removed it now. Sorry!